query10.sql

```diff
@@ -4,4 +4,24 @@
 */
 
 -- Enter your SQL query here
-select ...
+
+CREATE EXTENSION postgis;
+
+CREATE TABLE stations_geom AS
+  select id, name,
+    ST_SetSRID(ST_MakePoint(longitude,latitude),4326) as geom
+  FROM station_status
+;
+
+CREATE INDEX stations_gix
+  ON stations_geom
+  USING GIST (geom)
+;
```

Comment:
It's generally a good idea to create an index on your geometry fields, but notice that it's not actually serving any purpose here, since you're creating a new `distances` table and using that for all your calculations.